oak_functions/loader/fuzz/fuzz_targets/wasm_invoke.rs

```diff
@@ -0,0 +1,115 @@
+//
+// Copyright 2021 The Project Oak Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#![no_main]
+
+pub mod proto {
+    include!(concat!(env!("OUT_DIR"), "/loader.fuzz.abi_functions.rs"));
+}
+
+use crate::proto::{abi_function::Function, AbiFunctions};
+use arbitrary::{Arbitrary, Result, Unstructured};
+use libfuzzer_sys::fuzz_target;
+use oak_functions_abi::proto::Request;
+use oak_functions_loader::{
+    grpc::handle_request, logger::Logger, lookup::LookupData, server::WasmHandler,
+};
+use prost::Message;
+use std::sync::Arc;
+
+#[derive(Arbitrary, Debug)]
+enum AbiFunction {
```

Comment:
Could you impl To / From this to the proto version, instead of doing the translation inlined below?